repos/system_upgrade/common/actors/scansubscriptionmanagerinfo/tests/test_scansubscriptionmanagementinfo.py

```diff
@@ -0,0 +1,349 @@
+import logging
+import os
+from collections import namedtuple
+
+import pytest
+
+from leapp.libraries import stdlib
+from leapp.libraries.actor import scanrhsm
+from leapp.libraries.stdlib import api
+from leapp.libraries.common import mounting
+from leapp.libraries.common import repofileutils
+from leapp.libraries.common import rhsm
+from leapp.libraries.common.testutils import CurrentActorMocked, produce_mocked
+from leapp.models import RepositoryFile, RepositoryData
+
+
+RHSM_LIST_CONSUMED = ('subscription-manager', 'list', '--consumed')
+RHSM_STATUS = ('subscription-manager', 'status')
+RHSM_RELEASE = ('subscription-manager', 'release')
+RHSM_LIST_ENABLED_REPOS = ('subscription-manager', 'repos', '--list-enabled')
+
+RHSM_STATUS_OUTPUT_NONSATELLITE = '''
++-------------------------------------------+
+   System Status Details
++-------------------------------------------+
+Overall Status: Current
+
+System Purpose Status: Matched
+'''
+
+# Not the actual satelitte output (but it should suffice)
+RHSM_STATUS_OUTPUT_SATELLITE = '''
```

Comment:
This is about the SCA independent on satellite.
```suggestion
RHSM_STATUS_OUTPUT_SCA = '''
```